full-service/src/service/sync.rs

```diff
@@ -136,7 +145,24 @@ pub fn sync_all_accounts(
 
             continue;
         }
-        sync_account_next_chunk(ledger_db, conn, &account.id, logger)?;
+        let (num_received_txos, num_spent_txos) =
+            sync_account_next_chunk(ledger_db, conn, &account.id, logger)?;
+        if num_received_txos > 0 || num_spent_txos > 0 {
+            // call webhook
+            // FIXME move this somewhere and configured by config
+            // FIXME: should we be using rocket? or we've already got reqwest throughout
+            // anyway?
+            let client = Client::builder().gzip(true).use_rustls_tls().build()?;
```

Comment:
🚧 Under construction 🚧 

Plan is to have the client instantiated at the top - keeping everything together for now

Reply:
Went a different direction & eliminated